tools/crm_resource.c

```diff
@@ -674,6 +655,24 @@ cleanup_refresh_cb(const gchar *option_name, const gchar *optarg, gpointer data,
     return TRUE;
 }
 
+gboolean
+cmdline_config_cb(const gchar *option_name, const gchar *optarg, gpointer data,
```

Comment:
This merge (and the `agent_spec_cb()` creation in a later commit) end up disappearing later when we build `command_cb()`. I didn't know I was going to do that at the time, and I don't want to go back and undo these commits :)